aim/sdk/run.py

```diff
@@ -269,6 +278,20 @@ def __init__(self, run_hash: Optional[str] = None, *,
         ).subtree('seqs').subtree('chunks').subtree(self.hash)
 
         if not read_only:
+            if log_system_params:
```

Comment:
maybe we can put the `system_params` construction inside some utility function as well? as the `__init__` method of the `Run` class is pretty big and messy already by itself